pkg/pushsync/pushsync.go

```diff
@@ -94,12 +94,12 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 	defer func() {
 		if err != nil {
 			ps.metrics.TotalErrors.Inc()
+			ps.logger.Errorf("handler error; %v. resetting stream", err)
```

Comment:
`;` -> `:`

Reply:
actually, this logging is not necessary, since it will be logged by the p2p abstraction, right?

Reply:
Correct.